### PR TITLE
Source of truth

### PR DESCRIPTION
**Pull request by D-B-Hawk**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,8 +1,20 @@
 MONGODB_USERNAME=root
 MONGODB_PASSWORD=password
-MONGODB_HOST_TYPE=local
+MONGODB_HOST_TYPE=local # local | atlas
 MONGODB_HOST="localhost:27017"
+SERVER_PORT=8081
+K1_ACCESS_TOKEN=feedkray
+IS_CLUSTER_ZERO=`
```
Comment (johndietz) [inline .env.example]: @D-B-Hawk can you confirm this line and the set like it below - i'm not familiar with a single tick convention, feels like a bad find/replace

File: go.mod [hunk 2]
```diff
@@ -94,6 +94,7 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/bombsimon/logrusr/v2 v2.0.1 // indirect
 	github.com/bradleyfalzon/ghinstallation/v2 v2.1.0 // indirect
+	github.com/caarlos0/env/v10 v10.0.0 // indirect
```
Comment (jairoFernandez) [inline go.mod]: please run `go mod tidy`, the linter show this dependence should be direct, with go mod tidy you can fix this 👌🏼

File: internal/env/env.go [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+import (
+	env "github.com/caarlos0/env/v10"
+	"github.com/joho/godotenv"
+	log "github.com/sirupsen/logrus"
+)
+
+type Env struct {
+	ServerPort        int				`env:"SERVER_PORT" envDefault:"8081"`
+	K1AccessToken     string	  `env:"K1_ACCESS_TOKEN"`
```
Comment (jairoFernandez) [inline internal/env/env.go:11]: could you align this, are you using the golang extension or format on save vscode config?

File: internal/env/env.go [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+	}
+
+	environment := Env{}
+	err := env.Parse(&environment)
+	if err != nil {
+		return Env{}, err
+	}
+
+	return environment, nil
+}
```
Comment (jairoFernandez) [inline internal/env/env.go:46]: could you add some unit test like this example (this test is ✅ passing)
```go
// internal/env/env_test.go

package env

import (
	"os"
	"testing"
)

func TestEnv(t *testing.T) {
	os.Setenv("SERVER_PORT", "8081")
	os.Setenv("K1_ACCESS_TOKEN", "k1_access_token")
	os.Setenv("MONGODB_HOST", "mongodb_host")
	os.Setenv("MONGODB_HOST_TYPE", "mongodb_host_type")
	os.Setenv("MONGODB_USERNAME", "mongodb_username")
	os.Setenv("MONGODB_PASSWORD", "mongodb_password")
	os.Setenv("KUBEFIRST_VERSION", "development")
	os.Setenv("CLOUD_PROVIDER", "cloud_provider")
	os.Setenv("CLUSTER_ID", "cluster_id")
	os.Setenv("CLUSTER_TYPE", "cluster_type")
	os.Setenv("DOMAIN_NAME", "domain_name")
	os.Setenv("GIT_PROVIDER", "git_provider")
	os.Setenv("INSTALL_METHOD", "install_method")
	os.Setenv("KUBEFIRST_TEAM", "kubefirst_team")
	os.Setenv("KUBEFIRST_TEAM_INFO", "kubefirst_team_info")
	os.Setenv("AWS_REGION", "aws_region")
	os.Setenv("AWS_PROFILE", "aws_profile")
	os.Setenv("IS_CLUSTER_ZERO", "true")
	os.Setenv("IN_CLUSTER", "false")
	os.Setenv("ENTERPRISE_API_URL", "enterprise_api_url")

	defer func() {
		os.Unsetenv("SERVER_PORT")
		os.Unsetenv("K1_ACCESS_TOKEN")
		os.Unsetenv("MONGODB_HOST")
		os.Unsetenv("MONGODB_HOST_TYPE")
		os.Unsetenv("MONGODB_USERNAME")
		os.Unsetenv("MONGODB_PASSWORD")
		os.Unsetenv("KUBEFIRST_VERSION")
		os.Unsetenv("CLOUD_PROVIDER")
		os.Unsetenv("CLUSTER_ID")
		os.Unsetenv("CLUSTER_TYPE")
		os.Unsetenv("DOMAIN_NAME")
		os.Unsetenv("GIT_PROVIDER")
		os.Unsetenv("INSTALL_METHOD")
		os.Unsetenv("KUBEFIRST_TEAM")
		os.Unsetenv("KUBEFIRST_TEAM_INFO")
		os.Unsetenv("AWS_REGION")
		os.Unsetenv("AWS_PROFILE")
		os.Unsetenv("IS_CLUSTER_ZERO")
		os.Unsetenv("IN_CLUSTER")
		os.Unsetenv("ENTERPRISE_API_URL")
	}()

	env := Env{}
	env, err := GetEnv()
	if err != nil {
		t.Errorf("unexpected error: %v", err)
	}

	if env.ServerPort != 8081 {
		t.Errorf("expected ServerPort to be 8081, but got %d", env.ServerPort)
	}

	if env.K1AccessToken != "k1_access_token" {
		t.Errorf("expected K1AccessToken to be 'k1_access_token', but got '%s'", env.K1AccessToken)
	}

	if env.MongoDBHost != "mongodb_host" {
		t.Errorf("expected MongoDBHost to be 'mongodb_host', but got '%s'", env.MongoDBHost)
	}

	if env.MongoDBHostType != "mongodb_host_type" {
		t.Errorf("expected MongoDBHostType to be 'mongodb_host_type', but got '%s'", env.MongoDBHostType)
	}

	if env.MongoDBUsername != "mongodb_username" {
		t.Errorf("expected MongoDBUsername to be 'mongodb_username', but got '%s'", env.MongoDBUsername)
	}

	if env.MongoDBPassword != "mongodb_password" {
		t.Errorf("expected MongoDBPassword to be 'mongodb_password', but got '%s'", env.MongoDBPassword)
	}

	if env.KubefirstVersion != "development" {
		t.Errorf("expected KubefirstVersion to be 'development', but got '%s'", env.KubefirstVersion)
	}

	if env.CloudProvider != "cloud_provider" {
		t.Errorf("expected CloudProvider to be 'cloud_provider', but got '%s'", env.CloudProvider)
	}

	if env.ClusterId != "cluster_id" {
		t.Errorf("expected ClusterId to be 'cluster_id', but got '%s'", env.ClusterId)
	}

	if env.ClusterType != "cluster_type" {
		t.Errorf("expected ClusterType to be 'cluster_type', but got '%s'", env.ClusterType)
	}

	if env.DomainName != "domain_name" {
		t.Errorf("expected DomainName to be 'domain_name', but got '%s'", env.DomainName)
	}

	if env.GitProvider != "git_provider" {
		t.Errorf("expected GitProvider to be 'git_provider', but got '%s'", env.GitProvider)
	}

	if env.InstallMethod != "install_method" {
		t.Errorf("expected InstallMethod to be 'install_method', but got '%s'", env.InstallMethod)
	}

	if env.KubefirstTeam != "kubefirst_team" {
		t.Errorf("expected KubefirstTeam to be 'kubefirst_team', but got '%s'", env.KubefirstTeam)
	}

	if env.KubefirstTeamInfo != "kubefirst_team_info" {
		t.Errorf("expected KubefirstTeamInfo to be 'kubefirst_team_info', but got '%s'", env.KubefirstTeamInfo)
	}

	if env.AWSRegion != "aws_region" {
		t.Errorf("expected AWSRegion to be 'aws_region', but got '%s'", env.AWSRegion)
	}

	if env.AWSProfile != "aws_profile" {
		t.Errorf("expected AWSProfile to be 'aws_profile', but got '%s'", env.AWSProfile)
	}

	if env.IsClusterZero != true {
		t.Errorf("expected IsClusterZero to be true, but got false")
	}

	if env.InCluster != false {
		t.Errorf("expected InCluster to be false, but got true")
	}

	if env.EnterpriseApiUrl != "enterprise_api_url" {
		t.Errorf("expected EnterpriseApiUrl to be 'enterprise_api_url', but got '%s'", env.EnterpriseApiUrl)
	}
}

```

File: .env.example (hunk 1 above)
Comment (D-B-Hawk) [reply .env.example]: mistakes were made 🙃

File: internal/env/env.go (hunk 3 above)
Comment (D-B-Hawk) [reply internal/env/env.go:11]: this is something that is just showing up oddly. locally presents how it should 🤷

Comment (jairoFernandez) [reply internal/env/env.go:11]: your're right, however in local your branch looks:
CleanShot 2023-11-10 at 19 37 08

**Review by jairoFernandez — DISMISSED** [review]
great job! Maybe my approval isn't enough to merge, but here you go! 🚀

File: internal/env/env.go (hunk 3 above)
Comment (D-B-Hawk) [reply internal/env/env.go:11]: Screenshot 2023-11-10 at 4 41 06 PM

**Review by jarededwards — APPROVED** [review]

**Pull request merged**